### PR TITLE
[MBL-17551][Student][Teacher] Calendar selector redesign

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: See design in the ticket. The functionality should remain the same. Smoke test also calendar filters.

refs: MBL-17551
affects: Student, Teacher
release note: none

## Checklist

- [x] Tested in dark mode
- [x] Tested in light mode

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: libs/pandautils/src/main/java/com/instructure/pandautils/compose/composables/ListHeaderItem.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.instructure.pandautils.R
+
+@Composable
+fun ListHeaderItem(text: String, modifier: Modifier = Modifier) {
```
Comment (kristofnemere) [inline libs/pandautils/src/main/java/com/instructure/pandautils/compose/composables/ListHeaderItem.kt:33]: A preview might be useful for this

**Review by kristofnemere — APPROVED** [review]
QA+1

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Review by kristofnemere — APPROVED** [review]
QA+1

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**